### PR TITLE
Create a navigation view

### PR DESCRIPTION
**Pull request by simplyNoOne**
removed screen title and replaced with a custom navigation toolbar

### CODE REVIEW

File: PlannTer/Modules/MainScreenModule/MainScreenView.swift [hunk 1]
```diff
@@ -51,37 +51,56 @@ struct Tiles: View{
...
+        ScrollView(.horizontal, showsIndicators: false) {
+            VStack{
+                RoomScrollView(rooms: evenIndexedRooms, appendTile: appendTop, isTop: true)
+                RoomScrollView(rooms: oddIndexedRooms, appendTile: !appendTop, isTop: false)
+                
+            }
+        }.frame(width: 400)
+    }
+    private var evenIndexedRooms: [RoomModel] {
+           roomsList.enumerated().filter { $0.offset % 2 == 0 }.map { $0.element }
```
Comment (czaja307) [inline PlannTer/Modules/MainScreenModule/MainScreenView.swift:63]: that's cool

File: PlannTer/Modules/RoomModule/RoomDetailsView.swift [hunk 2]
```diff
@@ -1,25 +1,34 @@
...
-        NavigationView {
-            List(MockPlant.plants) { plant in
-                VStack(alignment: .leading) {
-                    Text(plant.name).font(.headline)
-                    
-                    HStack {
-                        Text("Water level: ")
-                        Text(String(plant.waterLevel))
+        VStack {
+            if /*@START_MENU_TOKEN@*/true/*@END_MENU_TOKEN@*/ {
```
Comment (czaja307) [inline PlannTer/Modules/RoomModule/RoomDetailsView.swift:9]: i may be missing something, but this one look suspicious at least
(did you insert some `if` statement by mistake or didn't finish editing it by any chance?)

File: PlannTer/Modules/RoomModule/RoomDetailsView.swift [hunk 3]
```diff
@@ -1,25 +1,34 @@
...
+                                Text(String(plant.waterLevel))
+                            }
+                        }
                     }
                 }
+                .frame(width: UIScreen.main.bounds.width)
+                .navigationBarBackButtonHidden(true)
+                .customToolbar(title: title, presentationMode: presentationMode)
+            } else {
+                /*@START_MENU_TOKEN@*/EmptyView()/*@END_MENU_TOKEN@*/
```
Comment (czaja307) [inline PlannTer/Modules/RoomModule/RoomDetailsView.swift:26]: same as above

**Review by czaja307 — COMMENTED** [review]
left a few comments, but remember i have no idea what i am doing writing a review in swiftUI :p

File: PlannTer/Modules/RoomModule/RoomDetailsView.swift (hunk 2 above)
Comment (simplyNoOne) [reply PlannTer/Modules/RoomModule/RoomDetailsView.swift:9]: it is suspisious, chat made it, it's irrelevant, as well - room details still need to be replaced with actual content

File: PlannTer/Modules/RoomModule/RoomDetailsView.swift (hunk 3 above)
Comment (simplyNoOne) [reply PlannTer/Modules/RoomModule/RoomDetailsView.swift:26]: same as above

**Review by czaja307 — APPROVED** [review]

**Pull request merged**